static/src/javascripts-legacy/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -29,11 +33,39 @@ define([
         laundromat,
 		regularsV2,
         acquisitionsEpicDesignVariationsV2,
+        acquisitionsEpicVsEpicAndEngagementBanner,
         askFourEarning,
         acquisitionsEpicArticle50Trigger,
         brexit];
 
+    var epicEngagementBannerTests = reduce(tests, function(out, test) {
+        var testInstance = new test();
+        if (testInstance.isEngagementBannerTest) {
+            out.push(testInstance)
+        }
+        return out;
+    }, []);
+
+    function getVariantIds(testInstance) {
+        return testInstance.variants.map(function(variant) {
+            return variant.id
+        })
+    }
+
+    var abTestClashData = tests.map(function(test) {
+        var testInstance = new test();
+        return {
+            name: testInstance.id,
+            variants: getVariantIds(testInstance)
```

Comment:
The original idea behind naming the variants was that some tests had some variants that violate the outbrain contact and some that don't. For example, in this test, the people who only see the banner are not violating outbrain.